app/practice/[situationId]/[detailId]/dial/call/hooks/useSpeechToText.ts

```
@@ -0,0 +1,157 @@
+			}
+		} else {
+			try {
+				recognitionRef.current.stop();
+			} catch (error) {
+				// aborted 에러는 무시 (이미 중단된 상태)
+				if (error instanceof Error && error.message.includes("aborted")) {
+					return;
+				}
+				console.warn("[STT] 중지 오류:", error);
+			}
+			setTranscript(""); // 텍스트 초기화
```

ESLint

[error] Error: Calling setState synchronously within an effect can trigger cascading renders

Effects are intended to synchronize state between React and external systems such as manually updating the DOM, state management libraries, or other platform APIs. In general, the body of an effect should do one or both of the following:
* Update external systems with the latest state from React.
* Subscribe for updates from some external system, calling setState in a callback function when external state changes.

Calling setState synchronously within an effect body causes cascading renders that can hurt performance, and is not recommended. (https://react.dev/learn/you-might-not-need-an-effect).

/home/jailuser/git/app/practice/[situationId]/[detailId]/dial/call/hooks/useSpeechToText.ts:146:4
  144 | 				console.warn("[STT] 중지 오류:", error);
  145 | 			}
> 146 | 			setTranscript(""); // 텍스트 초기화
      | 			^^^^^^^^^^^^^ Avoid calling setState() directly within an effect
  147 | 		}
  148 | 	}, [isRecording]);
  149 |

(react-hooks/set-state-in-effect)